pkgs/development/libraries/openssl/default.nix

```diff
@@ -163,6 +158,17 @@ let
         echo "Found an erroneous dependency on perl ^^^" >&2
         exit 1
       fi
+    '' + lib.optionalString (!withPerl) ''
+      # If withPerl is disabled, clear out any references to perl that remain in the bin output
+      # (after the postInstall hook above and fixup phase have run)
+      substituteInPlace $bin/bin/c_rehash --replace ${buildPackages.perl}/bin/perl "/usr/bin/env perl"
+      substituteInPlace $bin/bin/c_rehash --replace ${perl}/bin/perl "/usr/bin/env perl"
```

Comment:
https://github.com/NixOS/nixpkgs/blob/83e00a588d25210f91af0cc55aa96220b0fc73d1/pkgs/development/libraries/openssl/default.nix#L57-L59
Can it be addressed here?

Reply:
You mean by customizing the interpreter path fixup hook somehow? I don't know, I've never done that before. I did this stuff in the fixup phase to avoid disrupting the earlier code path where a `postInstall` hook does a substitution for the interpreter path fixup hook to pick up.

Reply:
By removing the last three lines. The `patchShebangs --build` should ignore perl after that.